opencti-platform/opencti-graphql/config/default.json

```diff
@@ -154,6 +154,10 @@
       "to_email": "no-reply@filigran.io",
       "max_list_size": 500
     },
+    "enrollment": {
+      "to_email": "no-reply@filigran.io",
```

Comment:
I think it's better to group in "xtm" block. Maybe xtmhub_to_email : if we send other email for xtm purpose we could re-use the same config.

About max_list_size I wonder if we shouldn't have one for the whole platform.... like "smtp.email_max_cc_size"